weblab/experiments/views.py

```diff
@@ -391,6 +416,7 @@ def get_success_url(self, *args, **kwargs):
         return reverse('experiments:versions', args=[self.get_object().experiment.id])
 
 
+
```

Comment:
Try to avoid adding extra blank lines :)

Reply:
noted - thanks